open-machine-learning-jupyter-book/data-science/data-visualization/visualizing-quantities.md

```diff
@@ -19,30 +19,30 @@ In this section you will explore how to use one of the many available Python lib
 
 ## Observe wingspan with matplotlib
 
-An excellent library to create both simple and sophisticated plots and charts of various kinds is [Matplotlib](https://matplotlib.org/stable/index.html). In general terms, the process of plotting data using these libraries includes identifying the parts of your dataframe that you want to target, performing any transforms on that data necessary, assigning its x and y axis values, deciding what kind of plot to show, and then showing the plot. Matplotlib offers a large variety of visualizations, but for this lesson, let's focus on the ones most appropriate for visualizing quantity: line charts, scatterplots, and bar plots.
+An excellent library to create both simple and sophisticated plots and charts of various kinds is [Matplotlib](https://matplotlib.org/stable/index.html). In general terms, the process of plotting data using these libraries includes identifying the parts of your dataframe that you want to target, performing any transforms on that data necessary, assigning its x and y axis values, deciding what kind of plot to show, and then showing the plot. Matplotlib offers a large variety of visualizations, but for this section, let's focus on the ones most appropriate for visualizing quantity: line charts, scatterplots, and bar plots.
 
 ```{note}
 Use the best chart to suit your data's structure and the story you want to tell.
-> - To analyze trends over time: line
-> - To compare values: bar, column, pie, scatterplot
-> - To show how parts relate to a whole: pie
-> - To show distribution of data: scatterplot, bar
-> - To show trends: line, column
-> - To show relationships between values: line, scatterplot, bubble
+- To analyze trends over time: line
+- To compare values: bar, column, pie, scatterplot
+- To show how parts relate to a whole: pie
+- To show distribution of data: scatterplot, bar
+- To show trends: line, column
+- To show relationships between values: line, scatterplot, bubble
 ```
 
 If you have a dataset and need to discover how much of a given item is included, one of the first tasks you have at hand will be to inspect its values.
 
-```{note}
+```{seealso}
 There are very good 'cheat sheets' available for Matplotlib [here](https://matplotlib.org/cheatsheets/cheatsheets.pdf).
 ```
 
 ## Build a line plot about bird wingspan values
 
-Create a new jupyter notebook file name as `let-us-learn-about-birds.ipynb` and add a cell.
+Open the file [let-us-learn-about-birds.ipynb](https://github.com/open-academy/machine-learning/tree/main/open-machine-learning-jupyter-book/assignments/data-science) with Jupyter Notebook and add a cell.
```

Comment:
I don't think your url link works here. Have you ever tried this link? Where it points to?

Actually, please use relative path, similar to how you refer to an image.